src/ui/icons/voice_search_dots.js

```diff
@@ -0,0 +1,95 @@
+import SVGIcon from './icon.js';
+export default new SVGIcon({
+  name: 'voice_search_dots',
+  viewBox: '0 0 33 27',
+  complexContents: `
+    <svg x="16.25" y="13.75" width="0.01" height="0.01" style ="overflow:visible" opacity="1" label="anchorPointContainer">
```

Comment:
if this works we could also update the other big icons in another item

Reply:
I found a rollup plugin that can handle the import, but it doesn't remove the whitespace. I think it makes sense to go with that since working with an SVG file is nicer than working with the template inline